core/templates/dev/head/pages/creator_dashboard/CreatorDashboard.js

```diff
@@ -126,10 +130,13 @@ oppia.controller('CreatorDashboard', [
 
     $scope.updatesGivenScreenWidth = function() {
       if ($scope.checkMobileView()) {
+        // Display is fixed to card in mobile view (no users choice)
         $scope.myExplorationsView = (
           constants.ALLOWED_CREATOR_DASHBOARD_DISPLAY_PREFS.CARD);
         $scope.publishText = EXP_PUBLISH_TEXTS.smText;
       } else {
+        // Displays as per the users choice card or list view
```

Comment:
"In non-mobile view, the display mode is set according to the user's preference."